### PR TITLE
changed formatting. No info updates

### PR DESCRIPTION
**Pull request by Ksenia-Galtsova**
Sorry, I had to fix inconsistent formatting in VisualCode, so the entire thing looks it was formatted...

### CODE REVIEW

File: channel-manager-api.md [hunk 1]
```diff
@@ -2,15 +2,24 @@
...
-* [Mews Inventory Update Modes](channel-manager-api.md#mews-inventory-update-modes)
-  * [Full Inventory Update Mode](channel-manager-api.md#full-inventory-update-mode)
-  * [Delta Inventory Update Mode](channel-manager-api.md#delta-inventory-update-mode)
-* [Expected Operations](channel-manager-api.md#expected-operations)
-  * [Update Prices](channel-manager-api.md#update-prices) \(required\)
-  * [Update Availability](channel-manager-api.md#update-availability) \(required\)
-  * [Update Restrictions](channel-manager-api.md#update-restrictions) \(required\)
-  * [Confirm Booking](channel-manager-api.md#confirm-booking) \(required\)
-  * [Change Notification](channel-manager-api.md#change-notification) \(optional\)
+* [Mews Inventory Update Modes
```
Comment (siroky) [inline channel-manager-api.md]: All the links are broken.

File: channel-manager-api.md [hunk 2]
```diff
@@ -406,7 +407,7 @@ or
 | `clientToken` | `string` | required | Client token of the channel manager. |
 | `connectionToken` | `string` | required | Token of a concrete connection. |
 | `channelManagerId` | `string` | required | Booking code. |
-| `error` | [`Error`](https://github.com/mews-systems/channel-manager-api/tree/da74c52f29ad04bf712acb397e311d8e5e8ba90b/general-api.md#error) | optional | If booking processing failed, holds the explanation. |
+| `error` | [`Error`](https: //github.com/mews-systems/channel-manager-api/tree/da74c52f29ad04bf712acb397e311d8e5e8ba90b/general-api.md#error) | optional | If booking processing failed, holds the explanation. |
```
Comment (patrikmiksovsky95) [inline channel-manager-api.md]:
```suggestion
| `error` | [`Error`](https://github.com/mews-systems/channel-manager-api/tree/da74c52f29ad04bf712acb397e311d8e5e8ba90b/general-api.md#error) | optional | If booking processing failed, holds the explanation. |
```

File: channel-manager-api.md [hunk 3]
```diff
@@ -450,7 +451,8 @@ or
 | `3` | Deactivated _In Mews, whole connection can be disabled._ |
 | `4` | Deleted |
 
-#### Response
+####
```
Comment (patrikmiksovsky95) [inline channel-manager-api.md]:
```suggestion
#### Response
```

File: channel-manager-api.md [hunk 4]
```diff
@@ -450,7 +451,8 @@ or
 | `3` | Deactivated _In Mews, whole connection can be disabled._ |
 | `4` | Deleted |
 
-#### Response
+#### 
 
-[Plain response](general-remarks.md#plain-response) is expected to determine whether the update was accepted or not.
+Response
```
Comment (patrikmiksovsky95) [inline channel-manager-api.md]:
```suggestion
```

File: channel-manager-api.md [hunk 5]
```diff
@@ -452,5 +453,5 @@ or
 
 #### Response
 
-[Plain response](general-remarks.md#plain-response) is expected to determine whether the update was accepted or not.
```
Comment (patrikmiksovsky95) [inline channel-manager-api.md]:
```suggestion
```

File: channel-manager-api.md [hunk 6]
```diff
@@ -453,4 +454,3 @@ or
 #### Response
 
 [Plain response](general-remarks.md#plain-response) is expected to determine whether the update was accepted or not.
```
Comment (patrikmiksovsky95) [inline channel-manager-api.md]:
```suggestion
[Plain response](general-remarks.md#plain-response) is expected to determine whether the update was accepted or not.

```

Comment (patrikmiksovsky95) [inline channel-manager-api.md]:
```suggestion
[Plain response](general-remarks.md#plain-response) is expected to determine whether the update was accepted or not.

```

**Pull request merged**